pkg/agent/flowexporter/types.go

```diff
@@ -40,10 +40,8 @@ type Connection struct {
 	// For invalid and closed connections: StopTime is the time when connection was updated last.
 	// For established connections: StopTime is latest time when it was polled.
 	StopTime time.Time
-	// IsActive flag helps in cleaning up connections when they are not in conntrack any module more.
-	IsActive bool
-	// DoExport flag helps in tagging connections that can be exported by Flow Exporter
-	DoExport           bool
+	// IsPresent flag helps in cleaning up connections when they are not in conntrack table anymore.
+	IsPresent          bool
```

Comment:
when is this consumed? I'm probably missing it but I can't find it right now

Reply:
Please see exporter.go. Changes of exporter.go is hidden.